09:29:18,592 ERROR com.virtualpairprogrammers.client.SimpleClientTest:26 - Some error occurred
09:29:19,839 DEBUG org.springframework.jdbc.core.JdbcTemplate:528 - Executing SQL update [CREATE TABLE CUSTOMER(CUSTOMER_ID VARCHAR(20), COMPANY_NAME VARCHAR(50), EMAIL VARCHAR(50), TELEPHONE VARCHAR(20), NOTES VARCHAR(255))]
09:29:19,843 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:110 - Fetching JDBC Connection from DataSource
09:29:20,413 TRACE hsqldb.db.HSQLDB6151AA7C37.ENGINE:? - log close size: 35
09:29:20,433  INFO hsqldb.db.HSQLDB6151AA7C37.ENGINE:? - Database closed
09:29:20,633 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:327 - Returning JDBC Connection to DataSource
09:29:20,745  INFO org.springframework.jdbc.support.SQLErrorCodesFactory:126 - SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
09:29:20,746 DEBUG org.springframework.jdbc.support.SQLErrorCodesFactory:199 - Looking up default SQLErrorCodes for DataSource [org.apache.commons.dbcp.BasicDataSource@29215f06]
09:29:20,749 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:110 - Fetching JDBC Connection from DataSource
09:29:20,751 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:327 - Returning JDBC Connection to DataSource
09:29:20,755 DEBUG org.springframework.jdbc.support.SQLErrorCodesFactory:217 - Database product name cached for DataSource [org.apache.commons.dbcp.BasicDataSource@29215f06]: name is 'HSQL Database Engine'
09:29:20,756 DEBUG org.springframework.jdbc.support.SQLErrorCodesFactory:175 - SQL error codes for 'HSQL Database Engine' found
09:29:20,757 DEBUG org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator:281 - Unable to translate SQLException with Error code '-5504', will now try the fallback translator
09:29:20,760 DEBUG org.springframework.jdbc.core.JdbcTemplate:528 - Executing SQL update [CREATE TABLE TBL_CALL(NOTES VARCHAR(255), TIME_AND_DATE DATE, CUSTOMER_ID VARCHAR(20))]
09:29:20,771 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:110 - Fetching JDBC Connection from DataSource
09:29:20,778 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:327 - Returning JDBC Connection to DataSource
09:29:20,779 DEBUG org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator:281 - Unable to translate SQLException with Error code '-5504', will now try the fallback translator
09:29:20,828 DEBUG org.springframework.jdbc.core.JdbcTemplate:528 - Executing SQL update [CREATE TABLE ACTION (ACTION_ID integer generated by default as identity (start with 1), DETAILS VARCHAR(255), COMPLETE BOOLEAN, OWNING_USER VARCHAR(20), REQUIRED_BY DATE)]
09:29:20,838 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:110 - Fetching JDBC Connection from DataSource
09:29:20,839 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:327 - Returning JDBC Connection to DataSource
09:29:20,839 DEBUG org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator:281 - Unable to translate SQLException with Error code '-5504', will now try the fallback translator
09:29:20,959 DEBUG org.springframework.jdbc.core.JdbcTemplate:906 - Executing prepared SQL update
09:29:20,960 DEBUG org.springframework.jdbc.core.JdbcTemplate:625 - Executing prepared SQL statement [INSERT INTO CUSTOMER (CUSTOMER_ID, COMPANY_NAME, EMAIL, TELEPHONE, NOTES) VALUES (?,?,?,?,?)]
09:29:20,960 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:110 - Fetching JDBC Connection from DataSource
09:29:20,982 DEBUG org.springframework.jdbc.core.JdbcTemplate:916 - SQL update affected 1 rows
09:29:20,983 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:327 - Returning JDBC Connection to DataSource
09:29:20,986 DEBUG org.springframework.jdbc.core.JdbcTemplate:691 - Executing prepared SQL query
09:29:20,987 DEBUG org.springframework.jdbc.core.JdbcTemplate:625 - Executing prepared SQL statement [SELECT * FROM CUSTOMER WHERE CUSTOMER_ID = ?]
09:29:20,996 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:110 - Fetching JDBC Connection from DataSource
09:29:21,009 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:327 - Returning JDBC Connection to DataSource
09:29:21,023 DEBUG org.springframework.jdbc.core.JdbcTemplate:691 - Executing prepared SQL query
09:29:21,024 DEBUG org.springframework.jdbc.core.JdbcTemplate:625 - Executing prepared SQL statement [SELECT ACTION_ID, DETAILS, COMPLETE, OWNING_USER, REQUIRED_BY FROM ACTION WHERE OWNING_USER=? AND COMPLETE=?]
09:29:21,024 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:110 - Fetching JDBC Connection from DataSource
09:29:21,025 DEBUG org.springframework.jdbc.datasource.DataSourceUtils:327 - Returning JDBC Connection to DataSource
09:29:21,039 TRACE hsqldb.db.HSQLDB6151AA7C37.ENGINE:? - log close size: 114
09:29:21,056  INFO hsqldb.db.HSQLDB6151AA7C37.ENGINE:? - Database closed
09:35:13,207 ERROR com.virtualpairprogrammers.client.SimpleClientTest:26 - Some error occurred
09:44:55,838 ERROR com.virtualpairprogrammers.client.SimpleClientTest:27 - Some error occurred
09:47:04,765 ERROR com.virtualpairprogrammers.client.SimpleClientTest:27 - Some error occurred
09:47:07,524 DEBUG com.virtualpairprogrammers.client.SimpleClientTest:39 - The number of records was 163
